components/ulp/cmake/toolchain-esp32-ulp.cmake

```diff
@@ -0,0 +1,17 @@
+# CMake toolchain file for ULP
+set(CMAKE_SYSTEM_NAME Generic)
+
+# Compiler is only used for preprocessing
+set(CMAKE_C_COMPILER "xtensa-esp32-elf-gcc")
+set(CMAKE_CXX_COMPILER "xtensa-esp32-elf-g++")
+
+set(CMAKE_ASM_COMPILER "esp32ulp-elf-as")
+set(CMAKE_OBJCOPY "esp32ulp-elf-objcopy")
+set(CMAKE_LINKER "esp32ulp-elf-ld")
+
```

Comment:
mone more NL

Reply:
This is IDF's code, not included in the Zephyr's build system